pegasus/migrations/105_populate_forms_hashed_email.rb

```diff
@@ -0,0 +1,30 @@
+require 'digest/md5'
+
+Sequel.migration do
+  up do
+    alter_table :forms do
+      add_column :hashed_email, String
+      add_index :hashed_email
+    end
+
+    # Update the existing data for HOC 2016 signups only to keep the
+    # count reasonable:
+    # mysql> select count(1) from forms where kind='HocSignup2016';
+    # +----------+
+    # | count(1) |
+    # +----------+
+    # |    48786 |
+    # +----------+
+    # 1 row in set (0.07 sec)
```

Comment:
Note that if you do go the script route, you should still keep do so also via migration for non-production environments, e.g., as in #9369.